config/src/main/kotlin/org/springframework/security/config/annotation/web/FormLoginDsl.kt

```diff
@@ -51,6 +51,15 @@ class FormLoginDsl {
 
     private var defaultSuccessUrlOption: Pair<String, Boolean>? = null
 
+    private var disabled = false
```

Comment:
Will you please update the copyright years in each of these files?

Reply:
Done